tests/model_serving/maas_billing/external_model/utils.py

```
@@ -0,0 +1,92 @@
+    name: str,
+    namespace: str,
+) -> dict[str, Any] | None:
+    """Look up an HTTPRoute by name/namespace. Returns the resource dict or None."""
+    try:
+        api = client.resources.get(
+            api_version="gateway.networking.k8s.io/v1",
+            kind="HTTPRoute",
+        )
+        route = api.get(name=name, namespace=namespace)
+        return route.to_dict() if route else None
+    except Exception:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)